.github/workflows/ci.yaml

```diff
@@ -56,6 +56,7 @@ jobs:
           mix local.rebar --force
           mix local.hex --force
           mix deps.get
+          mix deps.unlock --check-unused
```

Comment:
I am late to the party, but this should be a listing step of "Static code analysis"